lib/drivers/bayerContourNext.js

```diff
@@ -471,19 +495,21 @@ module.exports = function (config) {
       debug('in fetchData', data);
 
       function getOneRecordWithProgress(recnum, cb) {
-        var rec = data.rawrecords.shift();
         progress(100.0 * recnum / data.nrecs);
         setTimeout(function() {
-          getOneRecord(rec, data, cb);
+          getOneRecord(data, cb);
         }, 20);
       }
 
-      async.timesSeries(data.nrecs, getOneRecordWithProgress, function(err, result) {
+      // FIXME - We're adding +1 to skip the Patient Record (P)
```

Comment:
The first record that the Contour device gives back is a Patient record 'P'. The rest are data record types 'R'. I'll update the data structure to make it more obvious.